README.md

```diff
@@ -55,10 +55,18 @@ Parent is optional. The id field must be a string but can be called anything.
 client, _ := app.Firestore(ctx)
 fsc := firestorm.New(client, "ID", "")
 ```
-3. Optional. For optimal caching to work consider adding the CacheHandler
+##### Cache
+Firestorm supports adding a session cache to the context.
+The session cache only caches entities that are loaded within the same request.
+It does not cache searches (queries).
```

Comment:
You are right that queries do not load from the cache but entities are cached after they are loaded from firestore

Reply:
It is more clear if I write:
> Search queries are not cached (only the resulting entities).

?